src/linker/Linker/Driver.cs

```diff
@@ -674,11 +674,20 @@ public bool Run (ILogger customLogger = null)
 
 			try {
 				p.Process (context);
+			} catch (Exception ex) {
+				Console.Error.WriteLine ("The ILlinker exited with error:");
```

Comment:
I think we should use either "IL Linker" to refer to it in general, or "illink" when talking about the specific tool.

```suggestion
				Console.Error.WriteLine ("illink exited with error:");
```

Reply:
I used "IL Linker" to be consistent with the error messages.